scipy/integrate/_quadpack_py.py

```diff
@@ -615,11 +617,32 @@ def dblquad(func, a, b, gfun, hfun, args=(), epsabs=1.49e-8, epsrel=1.49e-8):
         an accuracy of ``abs(i-result) <= max(epsabs, epsrel*abs(i))``
         where ``i`` = inner integral of ``func(y, x)`` from ``gfun(x)``
         to ``hfun(x)``, and ``result`` is the numerical approximation.
-        See `epsrel` below.
+        See `epsrel` below. This arugment will be deprecated in a
+        future release, instead use the `opts` argument.
```

Comment:
Why not deprecate (and remove in 1.11)?

Reply:
I think that's what I meant to document. How does "This argument is deprecated and will be removed in a future release, instead use the `opts` argument" sound?

Reply:
There has been a lot of discussion about this recently, and we converged on a message like in https://github.com/scipy/scipy/pull/15708

Reply:
This has been fixed and is ready.